### DISCUSSION TITLE
Offset Wizzard Design

### DISCUSSION THREAD

**Discussion by Ghost1928563945** [ppy/osu#20947] [Ideas]
I think we should deign the offset wizard differently from the old osu because it looks bad. This is my concept. Please note that it is just a sketch and planing. Not a real design. 囧

https://www.figma.com/file/P1RipmpbSHJUdvCkcF6cSL/Osu-Offset-Wizard

**Comment by nagi-desuuu** [comment]
This should also be part of the first run setup, since osu is a rhythm game and the thing that a lot of players do when installing the game for the first time (I think) is to calibrate their offset right away.

**Reply by Ghost1928563945** [reply]
Yeah, I think it's a bit hard when hidden in the settings menu.

**Comment by bdach** [comment]
I for one have no idea as to what I'm looking at here on the attached figma.

**Reply by Ghost1928563945** [reply]
I don't know how I should reply. Would you mind explaining what you are talking about? Is it the preview on the right?

**Reply by ghost** [reply]
i don't get what "when you tap, a bar shows up" means, or how it should work

**Reply by bdach** [reply]
For a start I've got no idea what the horizontal bars are, what the "mode" drop-down is supposed to be doing, and what's the red "note" supposed to be saying. You can't just drop designs sans context and expect everyone to understand.

**Reply by Ghost1928563945** [reply]
> i don't get what "when you tap, a bar shows up" means, or how it should work

In Phigros when you tap the screen to the beat it creates a bar line in order to show you where you tap relative to the beat line/
